install/rpi_install_kalliope.sh

```diff
@@ -94,6 +94,56 @@ install_kalliope(){
     echo_green "Installing Kalliope...[OK]"
 }
 
+setup_pulseaudio(){
+    # Get the current user
+    current_user=$(logname)
+
+    # Check if default-server already exists in client.conf
+    if grep -Fxq "default-server = /var/run/pulse/native" /etc/pulse/client.conf; then
+        echo_yellow '"default-server = /var/run/pulse/native" already in /etc/pulse/client.conf'
+    else
+        # If not, we add it to /etc/pulse/client.conf
+        echo_green "Adding default-server to /etc/pulse/client.conf..[OK]"
+        echo "default-server = /var/run/pulse/native" | sudo tee -a /etc/pulse/client.conf >/dev/null
+    fi
+
+    # Check if autospawn already exists in client.conf
+    if grep -Fxq "autospawn = no" /etc/pulse/client.conf; then
+        echo_yellow '"autospawn = no" already in /etc/pulse/client.conf'
+    else
+        # If not, we add it to /etc/pulse/client.conf
+        echo_green "Adding 'autospawn = no' to /etc/pulse/client.conf..[OK]"
+        echo 'autospawn = no' | sudo tee -a /etc/pulse/client.conf >/dev/null
+    fi
+
+    # The Pi users needs to be in pulse-access group, check if already a member
+    if id -nG "$current_user" | grep -qw "pulse-access"; then
+        echo_yellow "$current_user is already a member of the group pulse-access"
+    else
+        # If not, then we add it to pulse-access
+        echo_green "Adding user $current_user to group pulse-access"
+        sudo usermod -a -G pulse-access $current_user
+    fi
+
+    if [[ -f "/etc/systemd/system/pulseaudio.service" ]]; then
+        # If the service already exists, we can skip this step
+        echo_green "Pulseaudio service already existing"
+    else
+        echo_yellow "Create pulseaudio service"
+        # Get the pulseaudio service file
+        wget -q https://raw.githubusercontent.com/kalliope-project/kalliope/master/install/files/pulseaudio.service
```

Comment:
instead of using wget here, could you maybe copy the file as we should have it from the clone. 
And so it will not depend of the "master" branch and can be tested by volunteer users from the dev branch.

Reply:
I was thinking the same, but I didn't come up with a proper solution but sure we already cloned it, so yes just copy it ^^ I'm gonna make the changes

Reply:
Done